scanapi/evaluators/string_evaluator.py

```diff
@@ -9,19 +9,50 @@
 
 
 class StringEvaluator:
+    """
+    Class that handles environment and custom variables evaluation.
+    It replaces every occurrence with ```${customVariable}```
+    or ```${ENV}``` pattern.
+    """
+
     variable_pattern = re.compile(
         r"(?P<something_before>\w*)(?P<start>\${)(?P<variable>[\w|-]*)(?P<end>})(?P<something_after>\w*)"
```

Comment:
@leonardomaier I know this is not related to your issue but the static analysis [pipeline](https://deepsource.io/gh/scanapi/scanapi/run/f348cef1-8ff0-48c9-b2dc-0f437972032e/python/FLK-E501) is failing. Can you please update the line with `# noqa` so that we can lint in part or maybe update `re.compile()` to fix the link? 

If it's too much of a hassle let me know, I can fix it in the same PR for you. So we merge this PR ASAP.

Reply:
@Pradhvan no problem, I'll be fixing it later 😉

Reply:
@Pradhvan I was reading Python's documentation looking for best way to fix this and I found this [topic](https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation)

Based on this example we would have something similar to:

```python
variable_pattern = re.compile(
    r"(?P<something_before>\w*)"
    r"(?P<start>\${)"
    r"(?P<variable>[\w|-]*)"
    r"(?P<end>})"
    r"(?P<something_after>\w*)"
)  # ${<variable>}
```
What do you think?

Reply:
@leonardomaier breaking the string literal down looks a much cleaner way to me. 

```Python
variable_pattern = re.compile(
    r"(?P<something_before>\w*)"
    r"(?P<start>\${)"
    r"(?P<variable>[\w|-]*)"
    r"(?P<end>})"
    r"(?P<something_after>\w*)"
)  # ${<variable>}
```

> Let me know what you think about it and once we agree with that I'll be sending these changes 😉

Go ahead. 👍🏾